pix2tex/dataset/render.py

```diff
@@ -7,112 +8,165 @@
 from tqdm.auto import tqdm
 import cv2
 import numpy as np
+from PIL import Image
+import traceback
+import subprocess
+import shlex
 
 
-def render_dataset(dataset: np.ndarray, names: np.ndarray, args):
-    '''Renders a list of tex equations
+def get_installed_fonts(tex_path: str):
+    cmd = "find %s -name *Math*.otf" % tex_path
+    process = subprocess.Popen(shlex.split(cmd),
+                               stdout=subprocess.PIPE,
+                               stderr=subprocess.PIPE,
+                               universal_newlines=True
+                               )
+    stdout, stderr = process.communicate()
+    if process.returncode != 0:
+        raise Exception(stderr)
+    fonts = [_.split(os.sep)[-1] for _ in stdout.split('\n')][:-1]
+    fonts.append("Latin Modern Math")
+    return fonts
 
+
+def render_dataset(dataset: np.ndarray, unrenders: np.ndarray, args):
+    '''Renders a list of tex equations
     Args:
         dataset (numpy.ndarray): List of equations
-        names (numpy.ndarray): List of integers of size `dataset` that give the name of the saved image
+        unrenders (numpy.ndarray): List of integers of size `dataset` that give the name of the saved image
```

Comment:
`unrenders` is a weird name. What does this mean? Maybe `toRender`?

Reply:
I think the name is a list of equation indexes that contains both to render and last failure rendered equation index.

some of these will never be rendered, so I call it un render. 
---
I'm poor at English, just think `name` is a kinda big range... so you can replace it what you like with a more specific name.

Reply:
Ok, `unrendered` seems like the word you were looking for.